### PR TITLE
console: spinner component integration (close #4183)

### LINKED ISSUE #4183: console: spinner component integration
Integrate <code>Spinner</code> component from the Design-System and replace all existing spinners.

reference ~ https://github.com/hasura/graphql-engine/issues/4136

### PR DESCRIPTION
**Pull request by lastnamearya**
### Description

<code>2</code> Design-System Integration PR

Before this PR, Please check and merge Icon PR

--------------------------------------------------------------------------------------------

- Integration of the <code>Spinner</code> component in console 
- 18 existing spinners replaced by the new spinner component from Design-System
- a new size <code>xl</code> added in the  spinner component and current size props name changed
  -  small to <code>sm</code>
  - large to <code>lg</code>

### Changelog

- [ ] `CHANGELOG.md` is updated with user-facing content relevant to this PR.

### Affected components

- [x] Console

### Related Issues

   #4183

#### Breaking changes

- [ ] No Breaking changes

### CODE REVIEW

File: console/src/components/Services/Data/TableBrowseRows/ViewRows.js [hunk 1]
```diff
@@ -29,9 +29,8 @@ import {
 import _push from '../push';
 import { ordinalColSort } from '../utils';
 import FilterQuery from './FilterQuery';
-import Spinner from '../../../Common/Spinner/Spinner';
 import Button from '../../../Common/Button/Button';
-import { Icon } from '../../../UIKit/atoms';
+import { Icon, Spinner } from '../../../UIKit/atoms';
```
Comment (beerose) [inline console/src/components/Services/Data/TableBrowseRows/ViewRows.js:33]: Position changed.

After:
browse-rows

Before:
browse-rows-before

File: console/src/components/Login/Login.js [hunk 2]
```diff
@@ -1,13 +1,15 @@
...
 import { verifyLogin } from './Actions';
 import { CLI_CONSOLE_MODE } from '../../constants';
 import { getAdminSecret } from '../Services/ApiExplorer/ApiRequest/utils';
+import { Spinner } from '../UIKit/atoms';
+import styles from './Login.scss';
 
-const styles = require('./Login.scss');
 const hasuraLogo = require('./blue-logo.svg');
 
 const Login = ({ dispatch }) => {
```
[the remark this answers is not in the document]
Comment (lastnamearya) [reply console/src/components/Login/Login.js:15]: It's fixed and yes the color is black now.

File: console/src/components/Services/Actions/Common/components/CloneTypeModal.js [hunk 3]
```diff
@@ -1,11 +1,12 @@
...
 import { isInputObjectType, isObjectType, isEnumType } from 'graphql';
+
 import { deriveExistingType } from '../utils';
 import Tooltip from './Tooltip';
-import styles from './Styles.scss';
 import { useIntrospectionSchema } from '../../../../Common/utils/graphqlUtils';
+import { Spinner } from '../../../../UIKit/atoms';
+import styles from './Styles.scss';
 
 const CloneType = ({ headers, toggleModal, handleClonedTypes }) => {
```
[the remark this answers is not in the document]
Comment (lastnamearya) [reply console/src/components/Services/Actions/Common/components/CloneTypeModal.js:11]: fixed.

File: console/src/components/Services/Actions/Codegen/Codegen.js [hunk 4]
```diff
@@ -1,17 +1,18 @@
...
 import { getPersistedDerivedMutation } from '../lsUtils';
-import Spinner from '../../../Common/Spinner/Spinner';
-import styles from '../Common/components/Styles.scss';
 import Button from '../../../Common/Button/Button';
 import CodeTabs from './CodeTabs';
 import DerivedFrom from './DerivedFrom';
+import { Spinner } from '../../../UIKit/atoms';
+import styles from '../Common/components/Styles.scss';
 
 const Codegen = ({ allActions, allTypes, currentAction }) => {
```
[the remark this answers is not in the document]
Comment (lastnamearya) [reply console/src/components/Services/Actions/Codegen/Codegen.js:17]: fixed.

File: console/src/components/Services/Data/TableBrowseRows/ViewRows.js (hunk 1 above)
Comment (lastnamearya) [reply console/src/components/Services/Data/TableBrowseRows/ViewRows.js:33]: fixed.

**Pull request closed without merging**